src/routes/events/+page.js

```diff
@@ -1,8 +1,8 @@
 import { error } from '@sveltejs/kit';
-import type { PageLoad } from './$types';
-import { getPages } from '../pageList';
+import { getPages } from '$lib/pageList';
 
-export async function load(): Promise<PageLoad> {
+/** @type {import('./$types').PageLoad} */
```

Comment:
I think you can get all the type benefits without this line: https://svelte.dev/blog/zero-config-type-safety
```suggestion
```

Reply:
I forgot about this, thanks!